issue-1185, further editing of plastics related terms

raissameyer opened the pull request:
PR addressing #1185, adding additional terms to the envoPlatics subset, updating 'nurdle', obsoleting 'plastic production compound' and 'manufactured plastic'.

pbuttigieg commented on src/envo/envo-edit.owl:
```diff
@@ -22555,16 +22555,13 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01000403> "t
-# Class: <http://purl.obolibrary.org/obo/ENVO_01000404> (manufactured plastic)
+# Class: <http://purl.obolibrary.org/obo/ENVO_01000404> (obsolete manufactured plastic)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://www.merriam-webster.com/dictionary/plastic") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01000404> "A (portion of) plastic is an (portion of) anthropogenic environmental material including any of numerous organic synthetic or processed materials which are primarily ...
```
Thanks @raissameyer looks like you used the obsoletion plugin - be sure there are no left over axioms on the class, but it looks like those are all gone.

raissameyer replied:
I double checked, and indeed, the axioms are gone

pbuttigieg commented on src/envo/envo-edit.owl:
```diff
@@ -39731,14 +39730,23 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03510038> "v
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03510039> "Plastics are produced by the conversion of natural products or by the synthesis of primary chemicals generally coming from oil, natural gas, or coal. Plasctic production compounds typically include elements such as carbon, hydrogen, oxygen, nitrogen, chlorine, and sulfur.")
-AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03510039> "plastic production compound"@en)
-SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03510039> <http://purl.obolibrary.org/obo/ENVO_00010483>)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03510039> "This term is ambigious, as it could refer to the compound the plastic is (supposed to be) made of, additives, contaminants, etc. Please instead directly refer to the compound of interest."^^xsd:string)
```
Agreed - this splits annotations

pbuttigieg commented on src/envo/envo-edit.owl:
```diff
@@ -40949,8 +40957,11 @@ AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.o
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105112> <https://orcid.org/0000-0002-8166-0486>)
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105112> <http://orcid.org/0000-0003-1072-8081>)
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105112> "2021-05-07T09:12:35Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_06105112> <http://aims.fao.org/aos/agrovoc/c_5998>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_06105112> <http://eurovoc.europa.eu/1837>)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105112> "manufactured plastic"@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105112> "envoPlastics")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105112> "envoPolar")
```
I'm not quite sure what the intent is here - please clarify

raissameyer replied:
Clarified in a call

pbuttigieg commented on src/envo/envo-edit.owl:
```diff
@@ -42215,6 +42226,7 @@ ObjectPropertyAssertion(<http://purl.obolibrary.org/obo/BFO_0000050> <https://ww
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_53243> "envoPlastics"@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_53550> "envoPlastics"@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_60034> "envoPlastics"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CHEBI_60160> "envoPlastics")
```
I note this doesn't have a language tag - intentional? As @cmungall noted in our Slack - these should actually be entered via the IRI editor, rather than literals anyway

raissameyer replied:
> I note this doesn't have a language tag - intentional?
I think this is due different people adding the subset tag. 

> As @cmungall noted in our Slack - these should actually be entered via the IRI editor, rather than literals anyway
I'll open a separate issue on this.

pbuttigieg reviewed (changes requested):
Thanks @raissameyer some questions on the content, but overall looking good.

raissameyer commented:
Thanks @pbuttigieg. I've addressed your questions above.

As I noticed that the obsoleted "plastic production compound" still had its original comment on it, I removed that.

Please let me know if you have any further questions or comments
